### ISSUE TITLE
Missing logs on logging server?

### ISSUE THREAD

**Issue by xeregin** [rcbops/ansible-lxc-rpc#130]
In looking through the logs (1.2 million entries) in our IAD lab, I'm not seeing anything for horizon, heat, or memcached. From what I understand, under normal situations, it may be normal not to have memcached logs, but there should definitely be horizon and heat logs. Is there something specific I should be searching for? Here is a readout of what I'm seeing:

```
Count   - Log Type
69120   - infra01_keystone
69120   - infra01_glance
17280   - infra01_nova_api_os_compute
17280   - infra01_nova_api_ec2
34562   - infra01_nova_api_metadata
69120   - infra01_cinder_api
82956   - logger1_logstash
4350    - dhclient
80665   - infra02_keystone
69112   - infra03_cinder_api
17278   - infra03_nova_api_os_compute
34556   - infra03_nova_api_metadata
69112   - infra03_glance
34557   - infra03_neutron_server
69113   - infra03_keystone
17278   - infra03_nova_api_ec2
69120   - infra02_cinder_api
17280   - infra02_nova_api_os_compute
69120   - infra02_glance
69256   - infra02_neutron_server
34560   - infra02_nova_api_metadata
17280   - infra02_nova_api_ec2
229906  - infra02_galera
4320    - cinder2_cinder_volumes
4320    - cinder1_cinder_volumes
1800    - CRON
368     - logger1_elasticsearch
2       - [u'_grokparsefailure']
18      - infra02_rabbit_mq
1       - infra03_neutron_agents
Analyzing: 1272810
Total: 1272810
```

**Comment by xeregin** [comment]
In the rsyslog container on infra01, /etc/rsyslog.d/51-openstack.conf has configuration entries for heat, but not for horizon. I've found heat logs in the heat container and horizon (apache) logs in the horizon container. However, on the rsyslog container, in /openstack/logs, there are heat logs, but no horizon logs (the directory is empty). On the logging server (from running a catchall query in the elasticsearch container), there are no entries for horizon and heat.

There seem to be no logs for memcached in its container (which is fine from my understanding).

**Comment by erikmwilson** [comment]
Is this issue fixed or do we need to add it to the known issues in the Release Notes?

**Comment by klevenstein** [comment]
@d34dh0r53 says it's fixed, so I think we're good.

**Comment by d34dh0r53** [comment]
yes this pulls both heat and horizon logs into the logging infrastructure.

**Comment by claco** [comment]
Fixed in master. Documented as known issue for 9.0.0. Needs a backport to 9.0.1

**Comment by mattt416** [comment]
Hi @d34dh0r53, I see nothing in logstash/kibana for horizon.  Should I?

**Comment by mancdaz** [comment]
When the PR went in, master was pre 9.0.0 so the fix is in 9.0.0 as well as current master
